core/arch/arm/dts/stm32mp135f-dk.dts

```diff
@@ -68,6 +68,51 @@
 	};
 };
 
+&etzpc {
+	st,decprot =
+		<&etzpc DECPROT(STM32MP1_ETZPC_ADC1_ID, DECPROT_NS_RW, DECPROT_UNLOCK)>,
```

Comment:
Remove `&etzpc` phandle references from `st,decprot` properties, here and in the other DTS files.

Reply:
Must be a mix on my side, sorry and good spot.